scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```
@@ -81,35 +39,44 @@ spec:
         trustedservices.intel.com/cache: pccs
     spec:
       tolerations:
-      - effect: NoSchedule
-        key: node-role.kubernetes.io/master
-        operator: Exists
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+          operator: Exists
       nodeSelector:
         kubernetes.io/hostname: ${PCCS_NODE}
       initContainers:
-       - name: init-seclabel
-         image: registry.access.redhat.com/ubi9/ubi:latest
-         command: ["sh", "-c", "chcon -Rt container_file_t /var/cache/pccs"]
-         volumeMounts:
-         - name: host-database
-           mountPath: /var/cache/pccs
-         securityContext:
-           runAsUser: 0
-           runAsGroup: 0
-           privileged: true  # Required for chcon to work on host files
+        - name: init-seclabel
+          image: registry.access.redhat.com/ubi9/ubi:latest
+          command: [ "sh", "-c", "chcon -Rt container_file_t /var/cache/pccs" ]
+          volumeMounts:
+            - name: host-database
+              mountPath: /var/cache/pccs
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
+            privileged: true  # Required for chcon to work on host files
       containers:
         - name: pccs
-          image: quay.io/openshift_sandboxed_containers/dcap/pccs:0.2.0
+          image: quay.io/openshift_sandboxed_containers/dcap/pccs:0.2.4
+          envFrom:
+            - secretRef:
+                name: pccs-secrets
+          env:
+            - name: "PCCS_HTTPS_PORT"
+              value: "8042"
+            - name: "PCCS_LOG_LEVEL"
+              value: "info"
+            - name: "CLUSTER_HTTPS_PROXY"
+              value: "${CLUSTER_HTTPS_PROXY}"
+            - name: "PCCS_FILL_MODE"
+              value: "REQ"
```

Not critical but we could keep the same value we used to have as the default:
```suggestion
              value: "LAZY"
```

---

Applied